deeplay/applications/timeseries/rnn.py

```diff
@@ -0,0 +1,239 @@
+from typing import Optional,overload,Sequence,List,Dict,Any,Literal,Union,Type
+from deeplay import DeeplayModule, Layer, LayerList, RecurrentBlock,MultiLayerPerceptron,LayerActivationNormalizationDropout
+
+import torch
+
+class RecurrentDropout(torch.nn.Module):
+    def __init__(self, p=0.0):
+        super(RecurrentDropout, self).__init__()
+        self.p = p
+        self.dropout = torch.nn.Dropout(p=self.p)
+
+    def forward(self, x):
+        if isinstance(x[0],torch.nn.utils.rnn.PackedSequence):
+            return torch.nn.utils.rnn.PackedSequence(self.dropout(x[0].data),x[0].batch_sizes),x[1]
+        return self.dropout(x[0]),x[1]
+
+class rnn(DeeplayModule):
```

Comment:
Agree with Carlo, maybe `RecurrentClassifier` or `SequenceClassifier` or `SeriesClassifier`. Try to keep the names as agent nouns